app.py

```diff
@@ -101,28 +102,42 @@ def get_database_configuration(
             f"Secret in wrong format found on {secret_name}")
 
 
-def try_connection(
+
+def mysql_connect(*args):
+    cnx = mysql.connect(*args)
+    return cnx
+
+def pg_connect(host, port, username, database, password, *_):
+    cnx = psycopg2.connect(
+            dbname=database,
+            user=username,
+            password=password,
+            host=host,
+            port=port
+    )
+    return cnx
+
+repo_connectors = {
+        "mysql": mysql_connect,
+        "pg": pg_connect,
```

Comment:
Please use the same naming conventions for repository names as defined in this other open source repository: https://github.com/cyralinc/terraform-cyral-sidecar-aws/blob/main/variables_sidecar.tf#L141
```suggestion
        "postgresql": pg_connect,
```